osclib/accept_command.py

```diff
@@ -279,6 +281,11 @@ def create_new_links(self, project, pkgname, oldspeclist):
                     source_file_save(self.api.apiurl, project, package, '_link', link)
         return True
 
+    def update_version_attribute(self, project, version):
+        version_attr = attribute_value_load(self.api.apiurl, project, 'ProductVersion')
```

Comment:
you don't have to be that careful with attributes, they don't trigger any rebuilds. The only thing to worry about is potentially polluting the project history. It doesn't hurt to be careful though :)